core/src/components/toggle/toggle.tsx

```diff
@@ -202,6 +202,7 @@ export class Toggle implements ComponentInterface {
           'toggle-switch-icon-checked': checked,
         }}
         icon={icon}
+        aria-hidden={true}
```

Comment:
```suggestion
        aria-hidden="true"
```
`aria-hidden` is an HTML attribute, so it expects `'true'` as a string not a boolean.